tests/infra/ccf.py

```diff
@@ -370,7 +371,10 @@ def check_for_service(self, node, status=ServiceStatus.OPEN):
                     return tables["ccf.service"]:get(0)"""
                 },
             )
-            current_status = rep.result["status"].decode()
+            if os.getenv("HTTP"):
```

Comment:
It's not really about HTTP, it's about JSON Text vs MsgPack.

Reply:
You're right. Fixed now.